js/main.js

```diff
@@ -0,0 +1,59 @@
+{
+  const addBtn = document.getElementById('add');
+  const todoTask = document.getElementById('txt');
+  
+  let num = 0;
+
+  function addTask() {
+    const tr = document.createElement('tr');
+    
+    id(tr);
+    comment(tr);
+    workingBtn(tr);
+    trashBtn(tr);
+    
+    document.querySelector('tbody').appendChild(tr);
+  }
+  
+  function id(tr) {
```

Comment:
```suggestion
  function appendIdTdElement(tr) {
```
メソッド名や関数名に `名詞` を使うのはあまりよろしくありません。
なぜかというとメソッドや関数は「振る舞い」があるものなので `名詞` では表現が十分にできないからです。
大抵の場合は「自動詞」または「他動詞 + 目的語」で命名することが多いです。